src/templates/rust-all/Makefile.tftpl

```diff
@@ -24,7 +26,7 @@ SGR0   := $(shell echo -e `tput sgr0`)
 
 # function with a generic template to run docker with the required values
 # Accepts $1 = command to run, $2 = additional flags (optional)
-docker_run = docker run \
+docker_run = docker pull -q $(BUILD_IMAGE_NAME):$(BUILD_IMAGE_TAG) && docker run \
```

Comment:
Will now always pull the image before running docker. This way we'll always running the latest build.